src/Plugin/EmulateElastic/AddTemplateHandler.php

```diff
@@ -47,7 +47,7 @@ public function run(): Task {
 				throw new \Exception('Cannot parse request');
 			}
 			$patterns = json_encode($request['index_patterns']);
-			foreach (['index_patterns', 'settings', 'mappings'] as $removeProp) {
+			foreach (['index_patterns'] as $removeProp) {
```

Comment:
I think we may get rid of this loop just due to us having only one element, to reduce nesting.

Reply:
Ok, I'll remove it